src/client/config/thema.ts

```diff
@@ -107,6 +108,12 @@ export const DocumentTitles: DocumentTitlesConfig = {
     `Vergunning | ${ThemaTitles.TOERISTISCHE_VERHUUR}`,
   [AppRoutes['TOERISTISCHE_VERHUUR/VERGUNNING/LIST']]:
     `Vergunningen | ${ThemaTitles.TOERISTISCHE_VERHUUR}`,
+  [AppRoutes['VAREN']]: `${ThemaTitles.VAREN} | overzicht`,
+  [AppRoutes['VAREN/LIST']]: `Vergunningen | ${ThemaTitles.VAREN}`,
```

Comment:
Je zou ook specifieker kunnen zijn (icm voorgestelde change aan de routes).
```suggestion
  [generatePath(AppRoutes['VAREN/LIST'], { kind: 'lopende-aanvragen' })]:
    `Lopende aanvragen | ${ThemaTitles.VAREN}`,
  [generatePath(AppRoutes['VAREN/LIST'], { kind: 'afgehandelde-aanvragen' })]:
    `Afgehandelde aanvragen | ${ThemaTitles.VAREN}`,
```